fuel-tx/src/transaction/types/mint.rs

```diff
@@ -73,7 +73,7 @@ pub struct Mint {
 impl crate::UniqueIdentifier for Mint {
     fn id(&self, chain_id: &ChainId) -> Bytes32 {
         if let Some(id) = self.cached_id() {
-            return id
+            return id;
```

Comment:
It would be nice if we skipped changes like this=D

Reply:
It's happening automatically by my formatter. I'm not sure what is different with my fmt configuration.